### PR TITLE
Account Button replaced with User's GitHub Profile Picture

### LINKED ISSUE #9691: [FEATURE] Using Users GitHub's profile photo instead of Account Button in Navbar
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

We can use user's github profile photo instead of displaying an Account button. It'll be more descriptive

### Proposed Solution

This is how it'll look after we replace the Account button with the user's profile pic

image

### Screenshots

_No response_

### Do you want to work on this issue?

Yes

### If you want to work on this issue...

_No response_

### PR DESCRIPTION
**Pull request by anirudhp26**
## Fixes Issue 

fixes #9691 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
Before
image
Before: Mobile Version
image

After
image
After: Mobile Version
image

### CODE REVIEW

File: components/navbar/Navbar.js [hunk 1]
```diff
@@ -116,7 +117,18 @@ export default function Navbar() {
...
           <NavLink
-            item={{ name: "Account", url: "/account/onboarding" }}
+            item={{
+              name: (
+                <Image
+                  className="flex-none rounded-full mx-2 bg-primary-low"
+                  width={42}
+                  height={42}
+                  src={`https://github.com/${session.username}.png`}
+                  alt="description of image"
```
Comment (akash19coder) [inline components/navbar/Navbar.js:127]: Would it be better if we add a bit more decriptive and accurate value to `alt` attribute in order to improve accessibility?
The following resource could be helpful:
[Resource for Image Alt text best practises](https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices#:~:text=The%20alt%20text%20needs%20to,for%20those%20using%20screen%20readers.)

**Review by sital002 — COMMENTED** [review]
Tested on:
Environment: Local Development
OS: Ubuntu
Browser: Firefox
works well but the image is too big and while hovering the borders are making it weird as well.
After changes:
image
image

File: components/navbar/Navbar.js (hunk 1 above)
Comment (anirudhp26) [reply components/navbar/Navbar.js:127]: Sure, I'll change it to a more descriptive text

**Comment by anirudhp26** [comment]
> Tested on: Environment: Local Development OS: Ubuntu Browser: Firefox works well but the image is too big and while hovering the borders are making it weird as well. After changes: image image

Then we can remove the borders or we can make them rounded, circle around the image.

I'll try different designs and let you know 👍

**Comment by anirudhp26** [comment]
Check this out, changed the size to 40, border is now rounded, and also the alt text is more relatable

image

**Comment by Abhishek-90** [comment]
> Check this out, changed the size to 40, border is now rounded, and also the alt text is more relatable
> 
> image

This does look better than the previous hover effect.

**Comment by eddiejaoude** [comment]
Please update the PR description with the latest screenshots

**Comment by anirudhp26** [comment]
Done

**Comment by eddiejaoude** [comment]
Thank you 👍  please include the response before/after also

File: components/navbar/Navbar.js [hunk 2]
```diff
@@ -115,10 +116,17 @@ export default function Navbar() {
 
       {session && (
         <>
-          <NavLink
-            item={{ name: "Account", url: "/account/onboarding" }}
-            setIsOpen={setIsOpen}
-          />
+          <Link href="/account/onboarding" aria-label="Account">
+            <div>
```
Comment (badrivlog) [inline components/navbar/Navbar.js:120]: Is this `div` needed?

Comment (badrivlog) [reply components/navbar/Navbar.js:120]: I'm not sure why the manage profile navigation link size is bigger than other link? 
IMG_20231102_235024


It's my suggestion, I think for consistent design we should use same size for all links, ensuring they look visually related and don't create confusion for users.

**Comment by badrivlog** [comment]
I'm not sure why the manage profile navigation link size is bigger than other links? 
IMG_20231102_235024


It's my suggestion, I think for consistent design we should use same size for all navigation links, ensuring they look visually related and don't create confusion for users.

File: components/navbar/Navbar.js (hunk 2 above)
Comment (anirudhp26) [reply components/navbar/Navbar.js:120]: yes without that div link wont work, don't know why

**Comment by anirudhp26** [comment]
> Thank you 👍 please include the mobile responsive before/after also

Surely, I'll include that

**Comment by anirudhp26** [comment]
> I'm not sure why the manage profile navigation link size is bigger than other links? IMG_20231102_235024
> 
> It's my suggestion, I think for consistent design we should use same size for all navigation links, ensuring they look visually related and don't create confusion for users.

but shrinking user's photo to that size won't look good according to me

File: components/navbar/Navbar.js (hunk 2 above)
Comment (eddiejaoude) [reply components/navbar/Navbar.js:120]: > yes without that div link wont work, don't know why

what does this mean? please explain

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you, looks good 👍 

On mobile navbar does not close after clicking on profile picture like other nav links do.

Please also see inline comment.

File: components/navbar/Navbar.js (hunk 2 above)
Comment (anirudhp26) [reply components/navbar/Navbar.js]: Well I removed the div, It seems to be working now, sorry for the inconvenience, I don't have any idea why it wasn't working before

**Comment by anirudhp26** [comment]
Working on the navbar closing issue

**Review by eddiejaoude — APPROVED** [review]
Looks good, thank you 👍

**Pull request merged**